### DISCUSSION TITLE
BREAKING CHANGE detected `@storybook/addon-interactions@8.2.0` removes `@storybook/test` module

### DISCUSSION THREAD

**Discussion by valpioner** [storybookjs/storybook#28516] [Help]
### Summary

Updating storybook to 8.2.0 broke our project.

Context:
We use `import { fn } from '@storybook/test';`
After we updated to 8.2.0, this module is not there any more.
After additional investigation, looks like `@storybook/addon-interactions@8.2.0` is responsible for that.

TS Error:
`Cannot find module '@storybook/test' or its corresponding type declarations.ts(2307)`

<img width="927" alt="image" src="https://github.com/storybookjs/storybook/assets/12964569/2e7e8162-03e5-4529-9296-50e2c07b2898">

**Comment by valentinpalkovic** [comment]
I will close this discussion in favor of the corresponding issue, which I have created:
https://github.com/storybookjs/storybook/issues/28517
